components/global/Button.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <content :is="type" :href="href" class="button btn" @click="onClick">
```

Comment:
```suggestion
  <content :is="type" :href="href" class="btn" @click="onClick">
```

Removing the unneeded `button` class.